stytch/b2b/organizations/types.go

```diff
@@ -484,7 +485,7 @@ type SCIMRegistration struct {
 	// ExternalID: The ID of the member given by the identity provider.
 	ExternalID string `json:"external_id,omitempty"`
 	// SCIMAttributes: An object for storing SCIM attributes brought over from the identity provider.
-	SCIMAttributes map[string]any `json:"scim_attributes,omitempty"`
+	SCIMAttributes *scim.SCIMAttributes `json:"scim_attributes,omitempty"`
```

Comment:
This is technically a breaking change, so we might need to make this a major version bump

Reply:
This is broken in it's current state. No one is using the scim client libraries so I think we're ok to minor bump as we're essentially just adding functionality ?

Reply:
Has SCIM been released? If so, I would say this is a MAJOR bump. But if it's truly broken (and has never worked), I'm okay calling it MINOR.

Reply:
Talked to @ollie-stytch today and I agree with him and @logan-stytch that it makes the most sense for this to be a minor bump - there's a breaking change in types, but the endpoint has never worked correctly and has no users.

@jennifer-stytch if that sounds good to you should we 🚢 ?

Reply:
sgtm! :shipit: